### PR TITLE
Added Solhint and Pylint

### LINKED ISSUE #12: Python and solidity linter GH action

### PR DESCRIPTION
**Pull request by anantasty**
# Linters for Solidity and Python
## Python 
Linting using [Pylint](https://pylint.org/)
Currently no linter configuration is used therefore linter uses default configuration
## Solidity
Uses [Solhint](https://protofire.github.io/solhint/)
Uses default configuration for solidity linting.
We also skip on error because current code with the default configurations does not pass linting.

Fixes #12

### CODE REVIEW

**Comment by theref** [comment]
Hi @anantasty, sorry for the delay in reviewing this. As an initial improvement, could you tidy up the git history a bit? Those 19 commits should probably be squashed down to only one. Thanks!

**Comment by anantasty** [comment]
@theref squashed the commits.

**Review by theref — APPROVED** [review]

File: scripts/deploy_subscription_manager.py [hunk 1]
```diff
@@ -30,5 +32,6 @@ def main(account_name : str = 'test'):
         calldata,
         {'from': deployer})
 
-    subscription_manager = Contract.from_abi("SubscriptionManager", transparent_proxy.address, subscription_manager_logic.abi, owner=None)
+    subscription_manager = Contract.from_abi(
+        "SubscriptionManager", transparent_proxy.address, subscription_manager_logic.abi, owner=None)
```
Comment (KPrasch) [inline scripts/deploy_subscription_manager.py:36]: Controversial opinion:  Linters make some lines more ugly and less readable to follow arbitrary line length rules.  This is an example of that.  Nonetheless this PR does look good to me.  Only suggestion I can make is potentially to just adjust the maximum line length.  whats the average screen resolution in 2022 anyways? 😄

**Review by KPrasch — APPROVED** [review]

File: scripts/deploy_subscription_manager.py (hunk 1 above)
Comment (theref) [reply scripts/deploy_subscription_manager.py:36]: We tend to use black (https://pypi.org/project/black/) everywhere anyway. So if people have `pre-commit` it should catch things before they hit this linter

**Pull request merged**